pkg/instrumentation/upgrade/upgrade.go

```diff
@@ -52,6 +58,28 @@ type InstrumentationUpgrade struct {
 	DefaultAutoInstApacheHttpd string
 	DefaultAutoInstNginx       string
 	DefaultAutoInstGo          string
+	defaultAnnotationToConfig  map[string]autoInstConfig
+}
+
+func NewInstrumentationUpgrade(client client.Client, recorder record.EventRecorder, cfg config.Config) *InstrumentationUpgrade {
```

Comment:
This function is only used in tests (that I can see).  Does it need included here?

Reply:
Sorry I forgot to put this in `main.go` after the change, it was supposed to move initialization of `InstrumentationUpgrade` to this function, will do it now